components/financial_assistance/app/domain/financial_assistance/operations/applications/aptc_csr_credit_eligibilities/renewals/renew.rb

```diff
@@ -100,7 +106,7 @@ def create_renewal_draft_application(application, validated_params)
                 if renewal_application.renewal_draft?
                   Success(renewal_application)
                 else
-                  Failure("Renewal Application Applicants Update or income_verification_extension required - #{renewal_application.hbx_id}")
+                  Failure("Renewal Application: (#{renewal_application.hbx_id}) failed with aasm_state: (#{renewal_application.aasm_state}), because: (#{@failure_reason || 'Unknown'})")
```

Comment:
More verbose failure message to provide details around the cause behind a failed renewal beyond the aasm_state.